model/SearchEngine/Driver/Elasticsearch/ElasticSearchConfig.php

```diff
@@ -44,7 +44,35 @@ public function __construct(ServiceOptionsInterface $serviceOptions)
 
     public function getHosts(): array
     {
-        return $this->serviceOptions->get(self::class, self::OPTION_HOSTS);
+        $hosts = [];
```

Comment:
We are repeating the same code several times. Can we please create a private function for that?

```
private function getFirstHost(): ?array 
{
    return current($this->serviceOptions->get(self::class, self::OPTION_HOSTS)) ?: null;
}
```